### PR TITLE
New readme.md files added inside level 2 folders

### PR DESCRIPTION
**Pull request by AnmolSirola**
## Description

New readme.md files were added inside level 2 folders and typos were fixed.

Fixes #<21>

## Type of change

- [x] Improve the readme of any project

## Domain of Contribution

- [x] Project Test on the Remix IDE

## Checklist ✓

- [x] I'm SSOC'22 contributor

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
Great work @AnmolSirola  .

File: Level2/Voting/README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+Once compiled successfully, click “Deploy and run transactions”. and "Deploy" it ,
+![deploy](https://user-images.githubusercontent.com/95535448/182790432-3c5bff42-5e2b-4d49-baf0-e6cdc2260922.png)
+
+Step 4:
+And here you can see the all the transactions and deployed contracts-
+![deployed contracts](https://user-images.githubusercontent.com/95535448/182790663-beb4c612-e4fd-4f44-8cd4-e3caa2f0a05d.png)
+
+Step 5:
+All the deployed contracts-
+1) partipant1_vote - it is a transact function (not payable)
```
Comment (Vikash-8090-Yadav) [inline Level2/Voting/README.md:35]: change the line  35  and below to 
1) partipant1_vote - it is used  to gave votes to the Participant 1 
2) partipant2_vote - it is used to   gave vote to the Participant 2 
3) declare_winner - declare's the winner of the poll (only declared by the owner )
6) parti1_cnt_vote - counts the vote of first participant  (only acessible by the owner ) 
7) parti2_cnt_vote - counts the vote of second  participant  (only acessible by the owner )

**Comment by Vikash-8090-Yadav** [comment]
@AnmolSirola   update this and all set for merge .. :smile:

**Comment by AnmolSirola** [comment]
Done Sir :)

**Comment by Vikash-8090-Yadav** [comment]
4) partipant1 - gives the address of first participant
5) partipant2 - give the address of second participant

include this point  also 
@AnmolSirola

**Comment by AnmolSirola** [comment]
Done :)

File: Level2/Voting/README.md [hunk 2]
```diff
@@ -35,7 +35,10 @@ All the deployed contracts-
 1) partipant1_vote - it is used to gave votes to the Participant 1
 2) partipant2_vote - it is used to gave vote to the Participant 2
 3) declare_winner - declare's the winner of the poll (only declared by the owner )
-4) parti1_cnt_vote - counts the vote of first participant (only acessible by the owner )
-5) parti2_cnt_vote - counts the vote of second participant (only acessible by the owner )
+4) partipant1 - gives the address of first participant
+5) partipant2 - give the address of second participant
+6) parti1_cnt_vote - counts the vote of first participant (only acessible by the owner )
+7) parti2_cnt_vote - counts the vote of second participant (only acessible by the owner )
+
```
Comment (Vikash-8090-Yadav) [inline Level2/Voting/README.md:43]: 8)owner - This is used to check the owner address 

@AnmolSirola include this last point

**Comment by Vikash-8090-Yadav** [comment]
Great work @AnmolSirola

**Comment by AnmolSirola** [comment]
Thank You sir

**Pull request merged**